build_js.sh

```diff
@@ -53,6 +53,8 @@ cmake ../src \
 cmake --build . -- -j 4
 cmake --build . --target install -- -j 4
 
+cp -v Release/bin/hsim_bindings.* esp/bindings_js/
```

Comment:
We shouldn't need to do this ideally, something incorrect is configured in CMake. Can someone else tell me what cmake variable we don't have properly configured? @mosra @aclegg3 ?

Reply:
Related: https://github.com/facebookresearch/habitat-sim/issues/527

I'd say this would fix it but unfortunately have no way to check locally at the moment -- @Ram81 @Skylion007 can you try and confirm?

```patch
diff --git a/src/esp/bindings_js/CMakeLists.txt b/src/esp/bindings_js/CMakeLists.txt
index d9df462d..3da748a6 100644
--- a/src/esp/bindings_js/CMakeLists.txt
+++ b/src/esp/bindings_js/CMakeLists.txt
@@ -12,7 +12,12 @@ target_link_libraries(hsim_bindings
     sim
 )
 
-set_target_properties(hsim_bindings PROPERTIES LINK_FLAGS "--bind")
+set_target_properties(hsim_bindings PROPERTIES
+  LINK_FLAGS "--bind"
+  # Override the Magnum-set default which is in $<CONFIG>/bin or $<CONFIG>/lib
+  RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+  LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+  ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 
 # copy JS/HTML/CSS resources for WebGL build
 set(resources

```

Reply:
@mosra tested on my local. `hsim_bindings` are getting updated in the `build_js/esp/bindings_js/`